core/tests/puppeteer-acceptance-tests/spec/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.spec.js

```diff
@@ -60,10 +60,6 @@ describe('Blog Admin', function() {
       await superAdmin.expectUserNotToHaveRole(
         'guestUsr2', ROLE_BLOG_POST_EDITOR);
 
-      await blogAdmin.expectTagToNotExistInBlogTags('Test_Tag');
```

Comment:
Does the functionality for the blog admin to do this still exist? What does their admin page look like now?

Reply:
I'm not sure if this existed before the changes as well. Have a look at the before and after changes:

before:
![Screenshot from 2023-05-24 16-35-55](https://github.com/oppia/oppia/assets/44300735/c8115424-29a6-4b9b-ae9e-6044f4266105)

after:
![Screenshot from 2023-05-24 16-34-45](https://github.com/oppia/oppia/assets/44300735/1fe28c61-2bdd-4775-97de-2a808d606720)

As we are no longer using the blog tags config property I have removed it.
I have also updated the description with the screenshots

Thanks

Reply:
This all seems good. Thanks!

Reply:
Just to be sure, Are we removing the tag functionality, or are we giving users the flexibility to create any tag?

Reply:
Hey @gp201, We are not removing the tag functionality instead now we have a fixed set of tags that will be available for the users to select from. We have a constant defined in `constants.ts` by the name `LIST_OF_DEFAULT_TAGS_FOR_BLOG_POST` and if in the future we plan to introduce some new tags we will simply be updating the value of LIST_OF_DEFAULT_TAGS_FOR_BLOG_POST.

Thanks